newclient/src/core/components/ui/AppStateDiff.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+    <v-dialog :hide-overlay="true" v-model="showDialog" justify-end width="300" origin="right">
+        <v-btn slot="activator" class="more-btn" color="white" fab depressed>
+            <p class="info--text title pb-2">...</p>
+        </v-btn>
+        <v-card class="overflow-hidden">
+            <v-card-title class="pb-0">
```

Comment:
can you change v-card-title to v-card text , the font is a little too large

Reply:
done!